torch/nn/functional.py

```diff
@@ -550,6 +550,19 @@ def bilinear(input1, input2, weight, bias=None):
         return Bilinear.apply(input1, input2, weight, bias)
 
 
+def embedding(input, weight, padding_idx=None,
+              max_norm=None, norm_type=2, scale_grad_by_freq=False,
+              sparse=False):
+    assert not sparse, 'sparse not supported in nn.functional.embedding for now'
```

Comment:
Yeah, thats fair. Probably an artifact of when I wasnt sure how I would implement this.